.github/workflows/assets-validation.yaml

```diff
@@ -52,7 +52,7 @@ jobs:
         run: pip install -e $scripts_azureml_assets_dir
       
       - name: Validate assets
-        run: python -u $scripts_assets_dir/validate_assets.py -i "${{ github.event.inputs.asset_dirs || env.default_asset_dirs }}" -a $asset_config_filename -c "${{ steps.changed-files.outputs.all_modified_files }}" -n -I -C -b -t
+        run: python -u $scripts_assets_dir/validate_assets.py -i "${{ github.event.inputs.asset_dirs || env.default_asset_dirs }}" -a $asset_config_filename -c "${{ steps.changed-files.outputs.all_modified_files }}" -n -I -C -b -t -e
```

Comment:
At some point we'll need to add an external file that contains YAML config or something similar. There are a lot of switches now!